src/forms.py

```diff
@@ -1,37 +1,42 @@
 from flask_wtf import FlaskForm
-from wtforms import StringField, PasswordField, TextAreaField, SubmitField
-from wtforms.validators import Email, Length, InputRequired, DataRequired
+from wtforms import StringField, PasswordField, TextAreaField, SubmitField, BooleanField
+from wtforms.validators import Email, Length, InputRequired, DataRequired, EqualTo
+
 
-"""
-[VARIABLE] = [FIELD TYPE]('[LABEL]', [
-    validators=[VALIDATOR TYPE](message='[ERROR MESSAGE]')
-])
-"""
 class RegisterForm(FlaskForm):
     email = StringField('email', [
         InputRequired(),
         Email(message='Invalid email'),
-        Length(max=30, message='Your email is too long')
+        Length(max=60)
     ])
     password = PasswordField('password', [
         InputRequired(),
-        Length(min=8, max=20, message='Your password is either too short or too long.')
+        EqualTo('confirm', message='Passwords must match'),
+        Length(min=5, max=60, message='Your password is either too short or too long.')
+    ])
+    confirm = PasswordField('Repeat Password', [
+        InputRequired()
     ])
+    submit = SubmitField('Register')
+
 
 class LoginForm(FlaskForm):
-    email = StringField('email',  [
+    email = StringField('email', [
         InputRequired(),
         Email(message='Invalid email'),
-        Length(max=30, message='Your email is too long')
+        Length(max=60, message='Your email is too long')
     ])
     password = PasswordField('password', [
         InputRequired(),
-        Length(min=8, max=20, message='Your password is either too short or too long.')
+        Length(min=8, max=30, message='Your password is either too short or too long.')
```

Comment:
Should this match with password requirements of the registration? Line nr 15.

Reply:
👍